meta-facebook/yv35-hd/src/platform/plat_class.c

```diff
@@ -269,7 +279,34 @@ void init_platform_config()
 				printf("Unknown the 1OU card type, the voltage of ADC channel-6 is %fV\n",
 				       voltage);
 			} else if (_1ou_status.card_type == TYPE_1OU_RAINBOW_FALLS) {
```

Comment:
Please add an else statement at the end of the else if to ensure anything not handled by the if/else-if cases is caught and processed.

Reply:
An else statement will be added to handle all conditions.